consul/README.md

```diff
@@ -1,32 +1,143 @@
 # Consul Integration
 
-## Overview
+# Overview
 
-Get metrics from consul service in real time to:
+The Datadog Agent collects many metrics from Consul nodes, including those for:
 
-* Visualize and monitor consul states
-* Be notified about consul failovers and events.
+* Total Consul peers
+* Service health - for a given service, how many of its nodes are up, passing, warning, critical?
+* Node health - for a given node, how many of its services are up, passing, warning, critical?
+* Network coordinates - inter- and intra-datacenter latencies
 
-## Installation
+The _Consul_ Agent can provide further metrics via DogStatsD. These metrics are more related to the internal health of Consul itself, not to services which depend on Consul. There are metrics for:
```

Comment:
> (and maybe link to the consul telemetry docs from here?)

A link on the marketing pages for these integrations? Glancing around the pages now, I don't see links in any of them.